### ISSUE TITLE
Feature - Auto connect using WiFi configuration file on SD card

### ISSUE THREAD

**Issue by lenisko** [Filipowicz251/mijia-1080P-hacks#36]
No phone assistance would be needed.
Simply check if there's change in `wpa_supplicant.conf` file on SD card (it's not yet there) if so just use this config and don't parse nvram things

I'll make it myself soon if no-one else pushes it.

**Comment by willthrom** [comment]
This was one of the old task I wanted to add.. Just lack of time.

Feel free to add it or just include it as an option in the midgard.ini to not have to many config files

**Comment by rmacian** [comment]
Could this help  me to connect to my 5Ghz wifi ? I can connect to 2.4Ghz on my Asus AC-68 Router but not to the 5ghz band

**Comment by BrendonKoz** [comment]
No, this is to help the camera to **remember** its WiFi settings. If it can't natively connect to (1)your router, or (2)a certain WiFi frequency, this fix won't help with that.

**Comment by willthrom** [comment]
The camera connect to 5Ghz without any problem, but just try  different channels.

I added the option in the midgard.ini files for the WIFI parameters, but haven´t implement the logic.
it is just calling nvram command to set them.... no time now, but feel free to implement it .

**Comment by carlchan** [comment]
just found this repo.  any hint on the nvram commands I should look at so I can try?

**Comment by santiacho** [comment]
I'm having problems to re-pairing the camera to Mi Home app, my reset button stoped working (some time before the hack) It seems a hardware problem.

Does anybody know how could I reset the camera via script and/or command? 

I need to delete previous wifi settings and start again but I do not know what else to try.

Thanks.
